src/k8s/cmd/k8sd_members/k8sd_members.go

```diff
@@ -0,0 +1,314 @@
+package k8sd_members
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	cmdutil "github.com/canonical/k8s/cmd/util"
+	"github.com/canonical/k8s/pkg/log"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+	"gopkg.in/yaml.v2"
+
+
+	"github.com/canonical/lxd/shared"
+	cli "github.com/canonical/lxd/shared/cmd"
+	"github.com/canonical/lxd/shared/termios"
+	"github.com/canonical/microcluster/v2/client"
+	"github.com/canonical/microcluster/v2/cluster"
+	"github.com/canonical/microcluster/v2/microcluster"
+)
+
+// TODO: we're only handling the k8sd database for the time being. Shoud
+// we also recover the k8s-dqlite database? What if this instance has the
+// most recent k8sd database but not the most recent k8s-dqlite db?
```

Comment:
yes, we must account for k8s-dqlite as well. otherwise, the cluster will not have a working datastore